vbox_create.sh

```diff
@@ -166,7 +169,7 @@ function create_cluster_VMs {
   # Gather VirtualBox networks in use by bootstrap VM
   oifs="$IFS"
   IFS=$'\n'
-  bootstrap_interfaces=($($VBM showvminfo ${CLUSTER_PREFIX}bcpc-bootstrap \
+  bootstrap_interfaces=($($VBM showvminfo ${BOOTSTRAP_NAME} \
```

Comment:
Could `vbox_create.sh` look for this to be defined and throw an error instead so we know its exogenous? (E.g. using something akin to `${BOOSTRAP_NAME?Need a bootstrap name provided in $BOOTSTRAP_NAME please.}`)

Reply:
At the same time `vbox_create.sh` leaks VM_LIST into `tests/automated_install.sh`, however in this particular case, why do we care?

Reply:
Yes, we should probably have a similar check in [`automated_install.sh`](https://github.com/bloomberg/chef-bach/blob/98e70e3c3579e1e7dd95230bb04574c5c702fba5/tests/automated_install.sh#L62). However, let us keep the documentation up akin to [line 61](https://github.com/bloomberg/chef-bach/blob/98e70e3c3579e1e7dd95230bb04574c5c702fba5/tests/automated_install.sh#L61).

Reply:
since this is driven by tests/automated_install.sh I don't know why we want to invest all this time in making this correct.  It will be set there.  I am not looking to make this bullet proof, this would be a complete rewrite of many parts.  I am only trying to restore the functionality that was there before.

Reply:
@pu239ppy Yes, please add error handling and/or documentation. (It should be the error message plus one characters and/or a one line comment for a future code reader's sake.)